include/ruby/internal/core/rfile.h

```diff
@@ -25,7 +25,7 @@
 
 /* rb_io_t is in ruby/io.h.  The header file has historically not been included
  * into ruby/ruby.h.  We follow that tradition. */
```

Comment:
Maybe this comment needs updating?

Reply:
I assume you refer to `rb_io_t` -> `rb_io`?

Reply:
Oh, I also mentioned the two headers.